dtfs-central-api/src/types/db-models/utilisation-data.ts

```diff
@@ -12,15 +13,9 @@ export type UtilisationData = WithId<{
    */
   bankId: string;
   /**
-   * 1-indexed month of the start of the report period
-   * example: 6 (for June)
+   * Start and end data of the report period
```

Comment:
I think it may be worth giving an example here (also should this maybe say `date` instead of `data`?)